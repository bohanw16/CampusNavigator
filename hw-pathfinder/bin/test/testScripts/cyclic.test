# find path in a cyclic graph

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3

AddEdge g n1 n2 1.111
AddEdge g n2 n3 1.111
AddEdge g n3 n1 1.111

FindPath g n1 n1