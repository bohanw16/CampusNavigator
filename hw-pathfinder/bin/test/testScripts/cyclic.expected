# find path in a cyclic graph

created graph g

added node n1 to g
added node n2 to g
added node n3 to g

added edge 1.111 from n1 to n2 in g
added edge 1.111 from n2 to n3 in g
added edge 1.111 from n3 to n1 in g

path from n1 to n1:
total cost: 0.000